.ci/linux-steps.yaml

```diff
@@ -38,7 +43,7 @@ steps:
   displayName: 'Installing Protobuf'
 
 # Configure mlboard (CMake)
-- script: mkdir build && cd build && cmake ..
+- script: mkdir build && cd build && cmake -DCEREAL_INCLUDE_DIR=../../cereal-1.3.0/include/ ..
```

Comment:
Are you sure you are going out of the directory? The last `cd` command was for mlpack, and then you install mlpack. maybe you need to add `cd ../../` and then download mlboard?
I can not see any other issue other than bad path for cereal